frontend/app/routes/$lang+/_public+/apply+/$id+/personal-information.tsx

```diff
@@ -76,7 +78,7 @@ export async function loader({ context: { session }, params, request }: LoaderFu
     id: state.id,
     csrfToken,
     meta,
-    defaultState: state.personalInformation,
+    defaultState: { ...state.personalInformation, email: state.personalInformation?.email ?? state.communicationPreferences?.email, confirmEmail: state.personalInformation?.confirmEmail ?? state.communicationPreferences?.confirmEmail },
```

Comment:
Nitpick: put this line on multple lines for readability.

```suggestion
    defaultState: {
      ...state.personalInformation,
      email: state.personalInformation?.email ?? state.communicationPreferences?.email,
      confirmEmail: state.personalInformation?.confirmEmail ?? state.communicationPreferences?.confirmEmail,
    },
```